catalog/package-lock.json

```diff
@@ -454,7 +454,7 @@
     },
     "acorn-jsx": {
       "version": "3.0.1",
-      "resolved": "http://registry.npmjs.org/acorn-jsx/-/acorn-jsx-3.0.1.tgz",
+      "resolved": "https://registry.npmjs.org/acorn-jsx/-/acorn-jsx-3.0.1.tgz",
```

Comment:
@akarve we should probably sync our npm versions / configs to avoid these meaningless diffs

Reply:
OK, I'm on 6.5.0. now